backend/start-dev.sh

```
@@ -0,0 +1,71 @@
+
+# Check if .env file exists
+if [ ! -f .env ]; then
+    echo "⚠️  No .env file found. Creating from .env.example..."
+    cp .env.example .env
+    echo "✅ Created .env file. Please configure it if needed."
+    echo ""
+fi
+
+# Load environment variables
+if [ -f .env ]; then
+    export $(cat .env | grep -v '^#' | xargs)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)